components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -1059,6 +1070,19 @@ public static boolean isSelfSignUpEPAvailable() {
         return false;
     }
 
+    /**
+     * Returns whether the email based username is enabled or not.
+     *
+     * @return true if the EnableEmailUserName. False if it doesnt
+     */
+    public static boolean isEmailUsernameEnabled() {
+        String enableEmailUsernameProperty = ServerConfiguration.getInstance().getFirstProperty(ENABLE_EMAIL_USERNAME);
```

Comment:
add an empty line after the method declaration.

Reply:
Fixed in 6616ab613867c2e0e048b7ddbfc9e7ed7ffa8601